src/assets/stylesheets/base.scss

```diff
@@ -1,2 +1,3 @@
-@import 'assets/stylesheets/base_styles/sanitize';
+// @import 'assets/stylesheets/base_styles/sanitize';
```

Comment:
@patrickkim I did a bit of research on this, and the best reasoning to use sanitize is presented on their website: https://jonathantneal.github.io/sanitize.css/

If you still think we should change over to reset.css, I'm down to discuss.